src/main/resources/db/migration/V1_7__visit_id_createdate.sql

```diff
@@ -0,0 +1,4 @@
+
+alter table visit_id add column when_created timestamp with time zone not null default now();
```

Comment:
was the 'with time zone' a deliberate inclusion? Be great if it avoids problems ive seen in testing where midnight is suddenly 23:00 the previous day etc !

Reply:
No, I have no idea if it is needed but stole it from Pathfinder!